test-client-web-app/package.json

```diff
@@ -7,7 +7,7 @@
     "@emotion/react": "^11.13.5",
     "@emotion/styled": "^11.13.0",
     "@fontsource/roboto": "^5.1.0",
-    "@foxglove/ws-protocol": "^0.7.1",
+    "@foxglove/ws-protocol": "0.8.0",
```

Comment:
should this be a major version bump instead?

Reply:
🤷  I think when the major version is 0, then it's allowed to have breaking changes that increment the minor version